samples/bluetooth/peripheral_lbs/Kconfig.sysbuild

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config NRF_DEFAULT_BLUETOOTH
```

Comment:
I'm building with
```
west build -b nrf5340dk_nrf5340_cpuapp --sysbuild
```
and flashing with `west flash --recover`, but the netcore doesn't seem to boot at all.
Am I holding it wrong?

I'll attach the logs.
[build-53.log](https://github.com/nrfconnect/sdk-nrf/files/12546225/build-53.log)

The endpoint binding fails (prolly because the netcore is not up) and even after enabling `CONFIG_LOG=y` there is no output on the netcore UART.

Reply:
I don't use west for building but tried using cmake and it works for me:
```
cmake -GNinja -DBOARD=nrf5340dk_nrf5340_cpuapp -DAPP_DIR=.. /tmp/dd/zephyr/share/sysbuild/
...
[269/269] Linking C executable zephyr/zephyr.elf
Memory region         Used Size  Region Size  %age Used
           FLASH:      126716 B      1008 KB     12.28%
             RAM:       27388 B       448 KB      5.97%
        IDT_LIST:          0 GB         2 KB      0.00%
[11/20] Performing build step for 'hci_rpmsg'
[1/232] Preparing syscall dependency handling

[2/232] Generating include/generated/version.h
-- Zephyr version: 3.4.99 (/tmp/dd/zephyr), build: v3.3.99-ncs1-2925-g0b32fd2743f5
[222/232] Linking C executable zephyr/zephyr_pre0.elf

[226/232] Linking C executable zephyr/zephyr_pre1.elf

[232/232] Linking C executable zephyr/zephyr.elf
Memory region         Used Size  Region Size  %age Used
           FLASH:      182656 B       256 KB     69.68%
             RAM:       49172 B        64 KB     75.03%
           SRAM1:          0 GB        64 KB      0.00%
        IDT_LIST:          0 GB         2 KB      0.00%
[20/20] Generating merged_CPUNET.hex
...
west flash
-- west flash: rebuilding
[0/10] Performing build step for 'peripheral_lbs'
ninja: no work to do.
[1/10] Performing build step for 'hci_rpmsg'
ninja: no work to do.
[8/9] cd /tmp/dd/nrf/samples/bluetooth/peripheral_lbs/_AA && /usr/bin/cmake -E true
-- west flash: using runner nrfjprog
Using board 1050097030
-- runners.nrfjprog: Flashing file: /tmp/dd/nrf/samples/bluetooth/peripheral_lbs/_AA/merged_CPUNET.hex
[ #################### ]  10.722s | Erase file - Done erasing                                                          
[ #################### ]   1.208s | Program file - Done programming                                                    
[ #################### ]   1.230s | Verify file - Done verifying                                                       
Applying pin reset.
-- runners.nrfjprog: Board with serial number 1050097030 flashed successfully.
-- west flash: using runner nrfjprog
Using board 1050097030
-- runners.nrfjprog: Flashing file: /tmp/dd/nrf/samples/bluetooth/peripheral_lbs/_AA/merged.hex
[ #################### ]   3.968s | Erase file - Done erasing                                                          
[ #################### ]   0.856s | Program file - Done programming                                                    
[ #################### ]   0.854s | Verify file - Done verifying                                                       
Applying pin reset.
...
ttyACM0: *** Booting Zephyr OS build v3.3.99-ncs1-2925-g0b32fd2743f5 ***
ttyACM1: *** Booting Zephyr OS build v3.3.99-ncs1-2925-g0b32fd2743f5 ***
Starting Bluetooth Peripheral LBS example
I: 2 Sectors of 4096 bytes
I: alloc wra: 1, fd0
I: data wra: 1, 1c
I: HW Platform: Nordic Semiconductor (0x0002)
I: HW Variant: nRF53x (0x0003)
I: Firmware: Standard Bluetooth controller (0x00) Version 220.17002 Build 1321500182
I: No ID address. App must call settings_load()
Bluetooth initialized
I: Identity: F6:39:AA:7E:BA:99 (random)
I: HCI: version 5.4 (0x0d) revision 0x20cb, manufacturer 0x0059
I: LMP: version 5.4 (0x0d) subver 0x20cb
Advertising successfully started
```

Reply:
oh gotcha. is this documented anywhere? Because a lot of use have been accustomed to just appending `--recover` when developing with nrf53.

Reply:
Zephyr bug: https://github.com/zephyrproject-rtos/zephyr/issues/50421

Reply:
> I don't use west for building

I think you should, at least as a check before submitting a build-system PR, as the documentation recommends using west for building and flashing.